pyppeteer/tracing.py

```diff
@@ -68,18 +69,18 @@ def __init__(self, client: CDPSession) -> None:
 
         :return: trace data as string.
         """
-        contentPromise = self._client._loop.create_future()
+        contentFuture = self._client._loop.create_future()
```

Comment:
I renamed promise suffix to future to represent the underlying structure better. IMO this doesn't really compromise readability when comparing to the js equivalent more than it improves upon the general readability.

Reply:
Sure, I agree with this - it's more explicit and the names aren't even exposed to public api.